martenbuild.cs

```diff
@@ -10,6 +12,9 @@ internal class MartenBuild
     {
         private const string BUILD_VERSION = "3.11.0";
 
+        private const string DockerConnectionString =
```

Comment:
**CONCERN:** That's minor, but wouldn't it be better to use the same config as in the pipeline tests? https://github.com/JasperFx/marten/pull/1463/files#diff-fec826feae04e51c0d94076385408bdcR36 It might be also good to use different database than default `postgres` as it might interfere with other user configuration.

Reply:
I'll make it use "marten_testing" as the database name.